py/client-ticking/build.gradle

```diff
@@ -26,15 +28,17 @@ deephavenDocker {
     networkName.set "pydeephaven-network-${randomSuffix}"
 }
 
-def buildPyClientTicking = Docker.registerDockerTask(project, 'pyClientTicking') {
+def buildPyClientTickingManyLinux = { wheelsSet, taskName -> Docker.registerDockerTask(project, taskName) {
```

Comment:
I'm not sure if we should want to build all of the wheels in a single task. It looks like the run command has a very coarse outer loop, so I'm not sure there's a big efficiency argument to be made?